packages/@pollyjs/core/src/-private/request.js

```diff
@@ -205,12 +198,16 @@ export default class PollyRequest extends HTTPBase {
     keys(NormalizeRequest).forEach(key => {
       if (this[key] && matchRequestsBy[key]) {
         identifiers[key] = NormalizeRequest[key](
-          key === 'body' ? this.serializedBody : this[key],
+          this[key],
```

Comment:
Pass the actual request body into the normalizer. We will serialize the the returned body after.